pkg/build/api/types.go

```diff
@@ -42,8 +49,8 @@ type Build struct {
 // BuildSpec encapsulates all the inputs necessary to represent a build.
 type BuildSpec struct {
 	// ServiceAccount is the name of the ServiceAccount to use to run the pod
-	// created by this build.
-	// The pod will be allowed to use secrets referenced by the ServiceAccount
+	// created by this build. The pod will be allowed to use secrets referenced
```

Comment:
@rhcarvalho first off, @mfojtik tagged the wrong person in his reply :)  and typoed 79 into 69.  He was intending to respond to @PI-Victor.

Now on to the questions....  I do pass in little changes with my PRs because i find it more efficient than a separate PR review process, especially if there's a chance my two PRs are going to end up conflicting w/ each other.

Does this set of changes meet that bar?  Probably.  it's a bit more than I think I would normally lump in with something else (since it's basically a massive reformatting) but as a reviewer I probably wouldn't reject it.

Bigger question:  do we actually want to reformat our code to 79 char limits.  I think we've been back and forth on that in various places and it's mixed, in our code.  I find that git does a decent job of showing you the specific word changes in paragraphs if that's all that changed, so line length doesn't tend to be a huge problem.  So i'm not a big fan of making it a rule.  @PI-Victor since you are in favor of this, can you check what is being done by other teams in other files?  I would prefer to follow their lead.